### PR TITLE
Fix pod slowness, and Vanilla kubernetes Scaled Endpoint Creation

### PR DESCRIPTION
**Pull request by phudtran**
This PR address issue #589.
This also fixes an issue for vanilla Kubernetes with the scaled endpoints not being created due to wrong naming

### CODE REVIEW

**Review by Hong-Chang — APPROVED** [review]
LGTM

File: mizar/daemon/interface_service.py [hunk 1]
```diff
@@ -135,14 +133,11 @@ def _ConsumeInterfaces(self, pod_name, cni_params):
...
-                self.queued_pods.remove(pod_name)
-            interfaces = self.interfaces.get(pod_name, [])
-            for interface in interfaces:
-                if interface.status == InterfaceStatus.queued:
-                    self._ProvisionInterface(interface, cni_params)
-                    interface.status = InterfaceStatus.consumed
+        interfaces = self.interfaces.get(pod_name, [])
+        for interface in interfaces:
+            if interface.status == InterfaceStatus.queued:
+                self._ProvisionInterface(interface, cni_params)
```
Comment (vinaykul) [inline mizar/daemon/interface_service.py:139]: Does this need to be synchronized? I'm just wondering what the intent of the original mutex was.

File: mizar/dp/mizar/workflows/builtins/services/create.py [hunk 2]
```diff
@@ -74,8 +74,10 @@ def run(self):
                             "VPC {} has no subnets to allocate service {}!".format(vpc_name, self.param.name))
                 net = net_opr.store.get_net(subnet_name)
         namespace = self.param.body['metadata']['namespace']
-        tenant = self.param.extra.get('tenant', '')
-        name = self.param.name + "-{}-{}".format(namespace, tenant)
+        name = self.param.name + "-{}".format(namespace)
+        if self.param.extra:
+            tenant = self.param.extra.get('tenant', '')
```
Comment (vinaykul) [inline mizar/dp/mizar/workflows/builtins/services/create.py:79]: Is this non empty only in the arktos scenario?

**Comment by vinaykul** [comment]
Overall LGTM. Just one minor clarification question to see if we didn't miss anything.

**Review by vinaykul — APPROVED** [review]

**Pull request merged**